widgetssdk/src/main/java/com/glia/widgets/view/floatingvisitorvideoview/FloatingVisitorVideoConstraintLayout.java

```diff
@@ -1,40 +1,35 @@
 package com.glia.widgets.view.floatingvisitorvideoview;
 
 import android.annotation.SuppressLint;
-import android.app.Activity;
 import android.content.Context;
 import android.util.AttributeSet;
 import android.view.LayoutInflater;
-import android.widget.FrameLayout;
 
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
+import androidx.constraintlayout.widget.ConstraintLayout;
 import androidx.core.util.Pair;
 
 import com.glia.androidsdk.comms.VisitorMediaState;
 import com.glia.widgets.R;
 import com.glia.widgets.di.Dependencies;
+import com.glia.widgets.helper.Utils;
 import com.glia.widgets.view.ViewHelpers;
 
-public class FloatingVisitorVideoCoordinatorLayout extends FrameLayout implements FloatingVisitorVideoContract.View {
+public class FloatingVisitorVideoConstraintLayout extends ConstraintLayout implements FloatingVisitorVideoContract.View {
```

Comment:
NIT: If you omit the view internal implementation (Coordinator, Constraint etc) from the name, then you don't have so many renaming changes next time the implementation is changed. Are there any benefits? 

Why not just `FloatingVisitorVideoView`?

Reply:
Ok, I see you already have a view with that name, but `FloatingVisitorVideoContainer` or smthng?

Reply:
renamed